src/components/App/style.scss

```diff
@@ -18,7 +18,27 @@ body {
   position: relative;
   margin: 0 0;
 }
-
+input[type="radio"] {
+ 
+  width:15px;
+  height:15px;
+  box-shadow: -4px 0 0 rgb(137, 49, 49); 
+  border-radius: 999px;
+  -webkit-appearance: inherit;
+  border:1px solid #999999;
+  position:relative;
+ 
+}
+input[type="radio"]:checked:before {
+  content:"";
+  position:absolute;
```

Comment:
po skończonej grze i zapisaniu wyników wraca się do widoku, w którym nie ma opcji wyboru trudności. 
I nie wiem, czy da się tu poczarować, ale można w ramach tego PR lub dorzucić do zadania z poprawkami scss wyświetlanie opcji poziomu trudności, bo na niektórych szerokościach pojawia się w dwóch liniach, z czego hard jest rozbite - kropka w jednej linii, a słowo w drugiej linii. no ale to znów drobnostka. przełączanie pytań działa super :) brawo za całość!

Reply:
dorzucone opcje wyboru trudności gry po skończonej grze